### DISCUSSION TITLE
Including other models rather than HF models in vLLM backend

### DISCUSSION THREAD

**Discussion by HalteroXHunter** [vllm-project/vllm#2593] [Q&A]
I wanted to ask regarding the configuration enabled by the vLLM Backend: https://github.com/triton-inference-server/vllm_backend

I wan to load a trained model in local that I don't have in the HF hub. Is there a possibility of loading models that are not in HF in the model.json file? https://github.com/vllm-project/vllm/blob/ee8217e5bee5860469204ee57077a91138c9af02/vllm/engine/arg_utils.py#L11

Or does the vLLM backend only support HF models?

Thanks in advance!

**Comment by CLopez021** [comment]
trying to run a local model but also only finding hf configs, any answers?
